2015-10-27.md

```diff
@@ -0,0 +1,32 @@
+Hi Tesselators!
+
+It's been a few weeks since the last This Week in Tessel, our apologies! We've been hard at work on the Tessel repos, building and growing the project.
+
+### Shipping Timeline
+After countless back and forths with our FCC testing counterparts, we have an end in sight. Seeed Studio expects us to have results from FCC testing back by Friday, November 27th. Once FCC certification is passed, sourcing the components will take up to 6 weeks. Then we can start producing in batches of 500/week. This puts the first shipments out in mid-January. Feel free to get in touch if the new date is an issue for you (support@tessel.io).
+
+### Get excited
+
+We've already built most of our Tessel Start experience for the Tessel 2 - [give yourself a tour](tessel.io/t2-start)! We've also released a set of preliminary Tessel 2 docs to [tessel.io/docs](https://tessel.io/docs/home).
+
+### Get involved
+
+We're also excited to have finished open sourcing literally all of our repos, including (at long last) our website, [tessel.io](https://github.com/tessel/tessel.io). Now, if you find something broken there you can file an issue or PR instead of reporting over Twitter.
+
+Interested in helping build Tessel 2? Two links for you:
+
+1. [Tessel's Slack](https://tessel-slack.herokuapp.com/) so you can ask us questions and get help getting started
+2. [The "Hardware Not Required" tag on our T2-CLI repo](https://github.com/tessel/t2-cli/labels/Hardware%20Not%20Required) where you can build functionality and get familiar with Tessel before your Tessel 2 ships
+
+### Welcome to the team
+
+Just taking a quick moment to call attention to all the people who have joined our growing volunteer contributor [team](https://github.com/tessel/project/blob/master/TEAM.md). We're happy to have you!
+
+New members from the last few months:
+
+[**Ravi Teja**](https://twitter.com/KTeza1) | [**Donovan Buck**](https://twitter.com/dtex) | [**Will Prater**](https://twitter.com/wprater) |  [**Dave Nugent**](https://twitter.com/drnugent) | [**Kevin Sidwar**](https://twitter.com/KevinSidwar) | [**Nicholas van de Walle**](https://twitter.com/mrn_ice) | [**Daniel Bunzendahl**](http://www.bunzendahl.com) | [**Amos King**](https://twitter.com/adkron) | [**Julian Cheal**](http://github.com/juliancheal)
+
+![](https://cloud.githubusercontent.com/assets/454690/10731290/945a114c-7bec-11e5-8a81-c61113c51b0b.png)
+
+With love,<br/>
+The Tessel Project
```

Comment:
it's hush hush but we give out a few subtle indicators that this project is just a massive AI